### DISCUSSION TITLE
Credit Card # when entered Doesnt Show Up

### DISCUSSION THREAD

**Discussion by AveryDBird** [woocommerce/woocommerce#50968] [Checkout Flow]
This is because Both the text and the enter area are white, there is no way to change this from the settings I have looked at.

**Comment by mikejolley** [comment]
This sounds like some styling conflicts in your theme—you'd have to fix this using custom CSS and either apply different text color or a background color on the inputs. If its a theme which declares support for WooCommerce you could ask the author.

Alternatively you could try dark mode inputs to see if that helps:

![Screenshot 2024-09-09 at 11 35 34](https://github.com/user-attachments/assets/a16a7f7c-e898-4332-bac6-7543ecef8656)
